pages/user.js

```diff
@@ -150,7 +150,10 @@ class User extends Component {
             <Button
                 type="primary"
                 size="small"
-                onClick={() => this.acceptChallengeRequest()}
+                onClick={() => {
+                    this.acceptChallengeRequest();
+                    notification.close(key);
+                }}
```

Comment:
create a method on the user class called `onButtonClick` and pass it as a property to `Button`
```
...
onClick={this.onButtonClick}
...
```
You might need to bind the function in the constructor.